### PR TITLE
Remove iOS 11.1 from data

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
To help reduce some confusion, this PR removes Safari iOS 11.1 as an option.  Safari 11.1 was introduced in iOS 11.3, yet our compatibility data contains iOS 11.1, which seems to present confusion about whether to use the Safari or iOS versions.

Depends on #4834.

### CODE REVIEW

File: html/elements/iframe.json [hunk 1]
```diff
@@ -1008,7 +1008,7 @@
                 "notes": "Currently only available on macOS High Sierra 10.13.4 beta, and in Safari Technology Preview 47+."
               },
               "safari_ios": {
-                "version_added": "11.1",
+                "version_added": "11.3",
                 "notes": "Currently only available on iOS 11.3 beta."
```
Comment (Elchi3) [inline html/elements/iframe.json]: Sorry for the drive-by change request, but can we remove this note and the Safari one? This kind of note doesn't age well. MDN authors: please don't add things like this.

Comment (vinyldarkscratch) [reply html/elements/iframe.json]: Removed!
